google/cloud/aiplatform/docker_utils/build.py

```diff
@@ -383,8 +412,8 @@ def make_dockerfile(
 def build_image(
     base_image: str,
     host_workdir: str,
-    main_script: str,
     output_image_name: str,
+    main_script: Optional[str] = None,
```

Comment:
Am I missing something? How is this `main_script` used? It doesn't seem to be passed in from `build_cpr_model`.

Reply:
The usage of `main_script` was removed in this change.